ciso_reports/csv.tmpl

```diff
@@ -0,0 +1,4 @@
+"Package","Version Installed","Vulnerability ID","Severity","FIX_AVAILABLE","IN_VERSION"
+{{- range .Matches}}
+"{{.Artifact.Name}}","{{.Artifact.Version}}","{{.Vulnerability.ID}}","{{.Vulnerability.Severity}}","{{.Vulnerability.Fix.State}}","{{.Vulnerability.Fix.Versions}}"
+{{- end}}
```

Comment:
small go templating script for grype to output CSV, more convenient than pure text files. grype has very little options on how to configure it on commandline unfortunately..